victor/src/layout/mod.rs

```diff
@@ -435,6 +442,37 @@ impl TextRun {
     }
 }
 
+/// https://drafts.csswg.org/css2/visuren.html#relative-positioning
+fn relative_adjustement(
+    style: &ComputedValues,
+    inline_size: Length,
+    block_size: Option<Length>,
+) -> Vec2<Length> {
+    let zero = Length::zero();
+    if !style.box_.position.is_relatively_positioned() {
+        return Vec2 {
```

Comment:
Nit for a future PR: we should add `impl Vec2<Length> { fn zero() -> Self {…} }`

Reply:
Done.